pkg/issuer/acme/dns/acmedns/acmedns.go

```diff
@@ -52,7 +52,10 @@ func NewDNSProvider(dns01Nameservers []string) (*DNSProvider, error) {
 // acme-dns server host is given in a string
 // credentials are stored in json in the given string
 func NewDNSProviderHostBytes(host string, accountJSON []byte, dns01Nameservers []string) (*DNSProvider, error) {
-	client := goacmedns.NewClient(host)
+	client, err := goacmedns.NewClient(host)
+	if err != nil {
+		return nil, err
```

Comment:
```suggestion
		return nil, fmt.Errorf("Error creating acme-dns client: %s", err)
```

Nit: To give an unlikely error some more context?